pkg/app/api/grpcapi/web_api.go

```diff
@@ -430,36 +430,60 @@ func (a *WebAPI) AddApplication(ctx context.Context, req *webservice.AddApplicat
 }
 
 func (a *WebAPI) UpdateApplication(ctx context.Context, req *webservice.UpdateApplicationRequest) (*webservice.UpdateApplicationResponse, error) {
-	claims, err := rpcauth.ExtractClaims(ctx)
-	if err != nil {
-		a.logger.Error("failed to authenticate the current user", zap.Error(err))
+	updater := func(app *model.Application) error {
+		app.Name = req.Name
+		app.EnvId = req.EnvId
+		app.PipedId = req.PipedId
+		app.Kind = req.Kind
+		app.CloudProvider = req.CloudProvider
+		app.Description = req.Description
```

Comment:
`UpdateApplicationRequest.Description` is added right before, isn't it? If so I think we can sneaky remove it. Or, still it's needed for somewhere?

Reply:
Nice catch! Let me remove it.

Reply:
Fixed it.